lib/common/command.js

```diff
@@ -112,7 +117,8 @@ module.exports = function(command, options){
       if (value.fromConfig)
         themes.fromConfig = true;
       return themes;
-    });
+    })
+    .option('--ignore-warnings', 'Set globs to be ignored by linter');
```

Comment:
Should be extended by normalize function like for `--theme` https://github.com/basisjs/basisjs-tools-build/blob/master/lib/common/command.js#L107-L115
Don't forget mark ignoreWarnings array in config https://github.com/basisjs/basisjs-tools-build/blob/master/lib/common/command.js#L54